### ISSUE TITLE
[Feature Request] Channel/Bus/Matrix Mute State Feedback available as a variable

### ISSUE THREAD

**Issue by ksurface** [bitfocus/companion-module-behringer-x32#116]
Would it be possible to make the mute state of a channel available as a variable in the same way that the fader level is available as a variable?

**Comment by chris-79** [comment]
Yes, please! 🙏  Would love to be able to save muted states to variables and recall them later
